api/app.py

```diff
@@ -21,17 +21,24 @@ def handle_preflight():
         return _build_cors_preflight_response()
 
 
+@app.after_request
+def handle_postflight(response):
```

Comment:
Do you not need to jsonify it too? or is it already jsonified at this point

Reply:
No idea, but it seems to work haha

Reply:
Do you wanna give it a try?